src/include/pbs_share.h

```diff
@@ -59,8 +59,9 @@
 #define FORMULA_ACCRUE_TYPE 	"accrue_type"
 
 /* Well known file to store job sorting formula */
-#define FORMULA_ATTR_PATH "server_priv/sched_formula"
-#define FORMULA_ATTR_PATH_SCHED "sched_priv/sched_formula"
+#define FORMULA_FILENAME "sched_formula"
+#define FORMULA_ATTR_PATH "server_priv/" FORMULA_FILENAME
+#define FORMULA_ATTR_PATH_SCHED "sched_priv/" FORMULA_FILENAME
```

Comment:
I don't think this is needed anymore, now that we are using sched_priv attribute